Autopilot/PathManager/Inc/CommWithAttitudeManager.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef GET_FROM_PATH_MANAGER_HPP
+#define GET_FROM_PATH_MANAGER_HPP
+
+//Commands to send to attitude manager.
+struct CommandsForAM{
+	float roll,pitch,yaw;	// commanded orientation (radians)
+	float airspeed;			// commanded airspeed (m/s)
+};
+
+//Data to receive from attitude manager.
+struct AttitudeData{
```

Comment:
I also want IMU stuff, so acc x,y,z and gyr x,y,z.

Of course, it s a struct so we can always add more stuff later if we want.